packages/datatrak-web/src/api/mutations/useResetPassword.ts

```diff
@@ -31,21 +31,21 @@ export const useResetPassword = (options?: {
       });
     },
     {
-      onError: () => {
-        if (options?.onError) options.onError();
+      onError: (error?) => {
+        if (options?.onError) options.onError(error);
       },
       onSettled: () => {
         if (options?.onSettled) options.onSettled();
       },
-      onSuccess: () => {
+      onSuccess: (response?) => {
```

Comment:
```suggestion
      onSuccess: (response?: { message: string }) => {
```

You can probably make a type at the start of this file for the response type too, to DRY things up here